arrowFn.js

```diff
@@ -0,0 +1,28 @@
+//1
+let toggle=false;
+
+const func=()=>{
```

Comment:
func is a very poor variable name. 
You have to try to write variable names in such a manner that code is readable and does not require comments.  So choose descriptive and readable names for variables and functions. 


the name of a function should be the answer to question `what does this function do ?`